src/Microsoft.PowerShell.Commands.Utility/commands/utility/MatchString.cs

```diff
@@ -93,6 +93,49 @@ public class MatchInfo
         /// <value>The text of the matching line.</value>
         public string Line { get; set; } = string.Empty;
 
+        /// <summary>
+        /// Gets or sets a value indicating whether the matched portion of the string is highlighted.
+        /// </summary>
+        /// <value>Whether the matched portion of the string is highlighted with asterisks.</value>
+        public bool Emphasize { get; set; }
```

Comment:
I believe this should be private. We don't use it publicly and have no need.

Reply:
Thanks, fixed.